assets/browser-fetch.js.t

```diff
@@ -33,6 +33,10 @@
 
     <%= moduleBody %>
 
+    if (!self.fetch) {
```

Comment:
This error should be changed to provide more actionable info if we agree on changes below.